.github/workflows/main.yml

```
@@ -69,112 +74,192 @@
+    runs-on: ubuntu-latest
+    if: always()
+    steps:
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ secrets.AWS_REGION }}
+
+      - name: Delete old ECR images
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2016:info:6:76: Expressions don't expand in single quotes, use double quotes for that

(shellcheck)